docs/source/reference/command_line.rst

```diff
@@ -346,10 +346,25 @@ Below is a list with all available subcommands.
       * Set a number of configuration options with sensible defaults
 
     Options:
-      --profile-name TEXT  Name of the profile. By default, a unique name starting with
-                           `presto` is automatically generated.  [default: (dynamic)]
-      --email TEXT         Email of the default user.  [default: aiida@localhost]
-      --help               Show this message and exit.
+      --profile-name TEXT             Name of the profile. By default, a unique name starting
+                                      with `presto` is automatically generated.  [default:
+                                      (dynamic)]
+      --email TEXT                    Email of the default user.  [default: aiida@localhost]
+      --use-postgres / --no-use-postgres
```

Comment:
Since this is opt-in, does it make sense to have `--no-use-postgres` variant here?

```suggestion
      --use-postgres
```

Reply:
True. It is just the standard approach for a feature switch. This will allow changing the default in the future, simply by setting `default=True` in the option definition. But I guess adding the off switch only when the default is toggled to True would be equivalent?

Reply:
Yeah. Or alternatively keep it functionally and just delete it from the docs here, to make it bit more clear what the default is.

Reply:
Docs are autogenerated, so to remove it, I actually have to remove the off-switch. But I think I will do that anyway